operator/v1/tests/ingresscontrollers.operator.openshift.io/SetEIPForNLBIngressController+IngressControllerLBSubnetsAWS.yaml

```diff
@@ -1,257 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
-name: "IngressController"
-crdName: ingresscontrollers.operator.openshift.io
-featureGates:
-- SetEIPForNLBIngressController
```

Comment:
These tests should probably still exist shouldn't they? What if we just remove the gating?

Reply:
I have moved the tests to operator/v1/tests/ingresscontrollers.operator.openshift.io/AAA_ungated.yaml as suggested by @everettraven but can rollback

Reply:
I prefer to have them separate so we can see which tests are related to which features, but, it's not a blocker

Reply:
done